crates/server/src/admin/service.rs

```diff
@@ -209,6 +219,20 @@ async fn install_application_handler(
     }
 }
 
+async fn install_dev_application_handler(
+    Extension(state): Extension<Arc<AdminState>>,
+    Json(req): Json<calimero_server_primitives::admin::InstallDevApplicationRequest>,
+) -> impl IntoResponse {
+    match state
+        .application_manager
+        .install_dev_application(req.application_id, &req.version, req.path)
```

Comment:
Does the user need to increase the version every time or he can use the same version but with changed wasm? Like hot reload.

Reply:
Wasm is linked so the next execution will use the new version.

Reply:
New version or new wasm?

Reply:
yeah, new wasm.. new version of the wasm, new update..